### ISSUE TITLE
misleading error: Please run actions/checkout before this action

### ISSUE THREAD

**Issue by avielb-navina** [tj-actions/changed-files#2254]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

Hey! 
when I tried to use this github action, i was always getting the error:
`Please run actions/checkout before this action`
even though I was following the documentation. 

eventually the root cause for this was even though I was in did in a git repository, i had to do fix as described below:

this is not working:
```
    - name: Checkout
      uses: actions/checkout@v4
      with:
        fetch-depth: 0
    - name: Get changed files
      id: changed-files
      uses: tj-actions/changed-files@v45
```

I was able to fix it only after doing as follows:
```
    - name: Checkout
      uses: actions/checkout@v4
      with:
        fetch-depth: 0
        clean: true
    - name: Config git
      run: |
        git config --global user.email "mail@example.com"
        git config --global user.name "avielb"
        git config --global --add safe.directory /__w/workspace/example-repo
    - name: Get changed files
      id: changed-files
      uses: tj-actions/changed-files@v45
```

### To Reproduce

.

### What OS are you seeing the problem on?

ubuntu-latest or ubuntu-22.04

### Expected behavior?

.

### Relevant log output

```shell
Please run actions/checkout before this action
```

### Has all relevant logs been included?

- [X] I've included all relevant logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jackton1** [comment]
Hi @avielb-navina. Unfortunately this issue seems to be missing any steps to reproduce the error or related logs require to provide a solution. 

Closing this issue for now since it doesn’t include the required information.

**Comment by jackton1** [comment]
Hi @avielb-navina, it is also worth noting that this issue is a duplicate of https://github.com/tj-actions/changed-files/issues/1948
